proposals/lip-0053.md

```diff
@@ -522,7 +522,8 @@ def execute(trs: Transaction) -> None:
     for ccmBytes in crossChainMessages:
         ccmID = sha256(ccmBytes)
         # Set ccmID (instead of the transaction ID) as default topic of this event.
-        defaultEventsTopic = ccmID
+        # The default topic is added automatically by emitEvent.
+        eventsDefaultTopic = ccmID
```

Comment:
Now I'm just wondering what's the scope of this change? I mean, when will the transaction ID be the default topic again? After the next event? After the for loop? After the function? Never? Maybe this should be clarified in the comment as well. Or we add an explicit `eventsDefaultTopic = transactionID` in the right place.
Depending on the answer, we may also be able to drop the 2nd `eventsDefaultTopic = ccmID` in this function.

Reply:
I explicitly added the transaction ID back as default event